### PR TITLE
Bump version to 0.20.1

### PR DESCRIPTION
**Pull request by JasonYeMSFT**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 0.20.1
+
+### Changed
+
```
Comment (nturinski) [inline CHANGELOG.md:6]: nit Usually we don't have a linebreak between header titles and the first bullet point.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-cosmosdb",
-    "version": "0.20.0",
```
Comment (nturinski) [inline package.json:3]: In the future, we should add the alpha suffix after we release.

**Review by nturinski — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (JasonYeMSFT) [reply package.json:3]: We did, this PR shows 0.20.0 because it branches off the commit where 0.20.0 is released.

File: CHANGELOG.md (hunk 1 above)
Comment (JasonYeMSFT) [reply CHANGELOG.md:6]: When I was working on Azure docs, the checking tool recommends adding a blank line before list of items. As far as I know, it's a more generally accepted markdown rule.

**Pull request merged**